debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java

```diff
@@ -46,6 +44,7 @@ public class DebeziumContainer extends GenericContainer<DebeziumContainer> {
     private static final String DEBEZIUM_NIGHTLY_TAG = "nightly";
 
     private static final int KAFKA_CONNECT_PORT = 8083;
+    private static final Duration DEBEZIUM_CONTAINER_STARTUP_TIMEOUT = Duration.ofMinutes(5);
```

Comment:
just to highlight it: this increases the timeout quite significantly. the old code waits only up to 1 minute.

Reply:
Is this high value really needed? This code is used for PR tests which might die late with such a high value. Would it be possible for your case to change the timeout to maybe 2 or 3 mins instead?

Reply:
I agree that 5 mins is a bit too much. I experimented with it and it seems that pull time is not counted to it. In that case I'd say 1 min should be enough. Instead of hard-coded value it is now configurable via globa wait timeout sysproperty.